gui/wxpython/datacatalog/catalog.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, parent, giface=None, id=wx.ID_ANY,
         # infobar manager for data catalog
         self.infoManager = DataCatalogInfoManager(infobar=self.infoBar,
                                                   giface=self.giface)
+        self.tree.showImportDataInfo.connect(self.showImportDataInfo())
```

Comment:
You are calling the `self.showImportDataInfo()` function here, you just need to pass it to connect just like with `ShowImportDataInfo` or at line 52.

Reply:
Has been corrected, now works as expected.